README.md

```diff
@@ -39,7 +39,7 @@ git clone <your fork>
 cp env-example .env
 bundle
 yarn
-bundle exec rails s
+bundle exec rspec spec
```

Comment:
You can probably just remove the spec bit of this. Running it as just Bundle exec rspec will run all of the specs anyway